ozaria/site/components/cinematic/common/CinematicCanvas.vue

```diff
@@ -92,8 +92,8 @@ export default {
         || document.documentElement.clientHeight
         || document.body.clientHeight, HEIGHT)
 
-      const width = this.width = Math.min(userWidth, WIDTH)
-      const height = this.height = Math.min(userWidth * CINEMATIC_ASPECT_RATIO, HEIGHT)
+      const height = this.height = Math.min(userWidth * CINEMATIC_ASPECT_RATIO, HEIGHT, userHeight)
+      const width = this.width = Math.min(this.height / CINEMATIC_ASPECT_RATIO)
```

Comment:
I think `Math.min` should not be required if this is a single value i.e. `this.height / CINEMATIC_ASPECT_RATIO`

Reply:
Great pickup. Can remove the `Math.min`.

Reply:
This let me simplify the logic in both the cutscene and cinematic. This was a great comment.